spec/services/publishers/ats_api/update_vacancy_service_spec.rb

```diff
@@ -41,20 +45,91 @@
 
   describe "#call" do
     context "when the update is successful" do
-      it "updates only the attributes that differ from the original vacancy" do
-        udpate_vacancy_service
+      it "updates the attributes that differ from the original vacancy" do
```

Comment:
I think vacancy.working_patterns stay the same because we provide the vacancies existing working_patterns in the params. Not sure if we want to change the description, or change the assertions to remove expect(vacancy.working_patterns).to eq(%w[full_time]), or if we want to set working_patterns in the params to "part_time" and assert that it changes to that?

Reply:
Good catch!